### DISCUSSION TITLE
Welcome to MG/SAIC HA Intergation Discussions

### DISCUSSION THREAD

**Discussion by ad-ha** [ad-ha/mg-saic-ha#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mstobie** [comment]


**Reply by townsmcp** [reply]
Please open an Issue ticket, grab some logs from HA and add to the issue ticket. I need logs to be able to create a car profile

**Comment by mstobie** [comment]


**Reply by townsmcp** [reply]
see issue ticket https://github.com/townsmcp/mg-saic-ha/issues/293.

**Comment by mstobie** [comment]
Hi James
It works!  Now reporting 100 kWh
Thank you
Michael Stobie
